table_filters/omero_filters.py

```diff
@@ -74,7 +68,11 @@ def get_script(request, script_name, conn):
         f = """(function filter(data, params) {
             if (isNaN(params.count) || params.count == '') return true;
             var table_data = %s;
-            var rowIndex = table_data.Well.indexOf(data.wellId);
+            if (data.wellId) {
+                var rowIndex = table_data.Well.indexOf(data.wellId);
+            } else {
+                var rowIndex = table_data.Image.indexOf(data.id);
```

Comment:
I recently found that the "Image" table column name doesn't match with "image" column name in ```populate_metadata.py``` https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/src/omero/util/populate_metadata.py#L264 which I had used as a reference for my [batch_roi_export_to_table](https://github.com/ome/training-scripts/pull/9/files#diff-80c2e3fd72b532331e94c17950a506b4) script.
So we should probably align this, or at least document the convention that parade is expecting.